packages/manager/modules/pci/src/projects/new/new.routing.js

```diff
@@ -82,22 +82,53 @@ export default /* @ngInject */ ($stateProvider) => {
               return $state.href('pci.projects');
           }
         },
-        shouldProcessChallenge: /* @ngInject */ (getCurrentStep, newProjectInfo) => () => {
+        shouldProcessChallenge: /* @ngInject */ (
+          $state,
+          atInternet,
+          getCurrentStep,
+          newProjectInfo,
+        ) => () => {
+          const page = `public-cloud::${$state.current.name.replace(/\./g, '::')}`;
```

Comment:
As this seems to be used quite often, maybe we could extract it to a function (at least the `replace` part)